src/main/java/core/basesyntax/service/Drawable.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.service;
+
+public interface Drawable {
```

Comment:
The `Drawable` interface should define a method for drawing, such as `void draw();`. This will ensure that any class implementing `Drawable` will provide an implementation for this method.